inspektor/commands/github.py

```diff
@@ -10,10 +10,56 @@
 # See LICENSE for more details.
 
 import logging
+import os
 
 from cliff.command import Command
 
-from inspektor.patch import GithubPatchChecker
+from inspektor import utils
+from inspektor.patch import PatchChecker
+
+
+class GithubPatchChecker(PatchChecker):
+
+    def __init__(self, args, patch=None, logger=logging.getLogger('')):
+        super(GithubPatchChecker, self).__init__(args=args, patch=patch,
+                                                 logger=logger)
+        self.github_id = args.gh_id
+        self.github_parent_project = args.parent_project
+        assert self.github_id is not None
+        assert self.github_parent_project is not None
+        self.patch = self._fetch_from_github()
+
+    def _get_github_repo_name(self):
+        return self.vcs.get_repo_name()
+
+    def _get_github_url(self):
+        """
+        Gets the correct github URL for the given project.
+        """
+        repo = self._get_github_repo_name()
+        return ("https://github.com/%s/%s/pull/%s.patch" %
```

Comment:
I don't use it anymore, and it's likely that nobody else uses it.

I can spend more time improving it in the direction you pointed out, but I'd like to do it at a later time. I feel like it can still be useful, and not having to use shell script is a huge plus in my book.